### PR TITLE
Fix/move functions for customize

### PR DESCRIPTION
**Pull request by LeratoKgomo**
i moved the functions to their own service file ...besides the pick images 
can you check if you can still change the profile picture and banner

### CODE REVIEW

File: lib/services/settings/customize_service.dart [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+          pictures.add(bannerDownloadUrl);
+          pictures.add(profileDownloadUrl);
+
+          customizeData.add(genres);
+          customizeData.add(age);
+          customizeData.add(interests);
+          customizeData.add(pictures);
+
+          
+          /*if (_isMounted) {
```
Comment (MonikaTheiss) [inline lib/services/settings/customize_service.dart:108]: Just a question, why did you comment this part out? I believe this is to fix a memory leak error that @pieryvdl fixed

**Review by MonikaTheiss — COMMENTED** [review]
Looks good on my side, added a comment , I will wait with my review until you have had the chance to respond to it.

**Review by pieryvdl — COMMENTED** [review]
Most of the changes look good! Same question as Monika on the setState that you commented out.

File: lib/services/settings/customize_service.dart (hunk 1 above)
Comment (LeratoKgomo) [reply lib/services/settings/customize_service.dart:108]: ya its still there but on the customize page

**Review by pieryvdl — APPROVED** [review]
Lerato explained that the code was kept in the customize page and not needed in the service.

**Review by MonikaTheiss — APPROVED** [review]
perfect , thank you!

**Pull request merged**